libraries/browser-tracker-core/src/tracker/out_queue.ts

```diff
@@ -69,6 +70,7 @@ export function OutQueueManager(
   postPath: string,
   bufferSize: number,
   maxPostBytes: number,
+  maxGetBytes: number | boolean,
```

Comment:
`numer | boolean` is an interesting choice. I think if you passed in `true` then this would break, or at least have unexpected behaviour. You'd end up doing `if (bytes >= true) {` on Line 240.

You could do `number | false` instead, or just `number` and have the behaviour of `0` (or less I guess) not applying a max bytes?

Reply:
Good point, I didn't consider the `true` case. Changed it to only accept numbers where 0 or less is considered as unlimited.